modules/javafx.controls/src/main/java/javafx/scene/control/Labeled.java

```diff
@@ -818,6 +828,37 @@ public final BooleanProperty mnemonicParsingProperty() {
         return mnemonicParsing;
     }
 
+    /**
+     * Indicates whether the text has been truncated
+     * because it cannot fit into the available width.
+     * <p>
+     * When truncated, the {@link #ellipsisStringProperty() ellipsisString}
+     * gets inserted in the place dictated by the
+     * {@link #textOverrun} property.
+     *
+     * @since 23
+     */
+    private ReadOnlyBooleanWrapper textTruncated;
+
+    public final ReadOnlyBooleanProperty textTruncatedProperty() {
+        return textTruncated().getReadOnlyProperty();
+    }
+
+    public final boolean isTextTruncated() {
+        return textTruncated().get();
+    }
+
+    private final void setTextTruncated(boolean on) {
+        textTruncated().set(on);
+    }
+
+    private ReadOnlyBooleanWrapper textTruncated() {
```

Comment:
Suggestion: rename this method to something like `textTruncatedImpl()` for clarity (as it is, the method, which is a writable property, shares the same name as the read-only property field, which could be confusing).

Reply:
Never mind. I missed that the types of the field and method are identical, so I withdraw my suggestion.